### ISSUE TITLE
npm package request deprecated 

### ISSUE THREAD

**Issue by jasperchua99** [filipedeschamps/rss-feed-emitter#224]
https://github.com/request/request/issues/3142 

this npm package seam to be deprecated, any reification to use another package?

**Comment by jasperchua99** [comment]
oic

**Comment by filipedeschamps** [comment]
You guys are right. After others projects and a recently born son my life has changed significantly. I've also published [a video on my channel](https://youtu.be/amdnXhAaA1M) explaining how I screwed up with my open source projects.

But what can I do to fix this? @TobiTenno you have `write` access to the repo I think, maybe you can start merging things to master by yourself, you are doing an amazing job. And I saw you made a PR to start using Github Actions. What do you think to remove all external dependencies (including Coverals)? Also, let's not auto-publish do `npm`. I prefer doing this manually.

**Comment by jasperchua99** [comment]
well im all ears to hears what yall are gonna do to make sure this project goes smoothly, for now tho, request is the only one bugging my npm to change to another file

**Comment by TobiTenno** [comment]
Yeah, I'll get started on that migration once I finish the 2 others (config spread and dedupe)

**Comment by TobiTenno** [comment]
at this point, i'm gonna have to lean on y'all, cause I can't merge my own stuff still, so if y'all can submit fixes, i can merge them

**Comment by sankemax** [comment]
@TobiTenno hey, if i create a PR, will you merge it?

**Comment by TobiTenno** [comment]
If it works, yes

**Comment by TobiTenno** [comment]
@sankemax would you be down to help me by basically resubmitting several of my old PRs that include things like reducing config spread and moving to gh actions? I can help manage more PRs and get more updated off of travis and deprecated deps

**Comment by TobiTenno** [comment]
@filipedeschamps we really need you to do those parts switched out on #216. it would be even better if you turned this into an org project and migrated it there, that would give you and i equal access to the configs and i'd be able to merge my own PRs without waiting for approvals, just for CI to merge

**Comment by filipedeschamps** [comment]
Done @TobiTenno 👍

**Comment by TobiTenno** [comment]
everything should be updated now, sorry for my slowness. i'm still hoping to update some other dependencies, but it'll mean breaking version changes, and i'm trying to do as few of those as possible

**Comment by TobiTenno** [comment]
i think the new version of feed-parser supports node-fetch or node's fetch directly, i'm working on updating those
